bitnami/elasticsearch/README.md

```diff
@@ -875,6 +808,10 @@ Find more information about how to deal with common errors related to Bitnami's
 
 ## Upgrading
 
+### To 19.0.0
+
+The new version of this chart is not longer sopported elasticsearch-curator, this repository has been deprecated.
```

Comment:
Please fix the typos here. 

I would suggest something like:
'This new version of the bitnami/elasticsearch chart no longer supports elasticsearch-curator. The bitnami/elasticsearch-curator images that have been released will continue being available, but they will no longer receive new releases.'

Reply:
I update the small comment about the deprecation of Elasticsearch-curator.